build.gradle

```diff
@@ -115,6 +116,12 @@ subprojects {
         testLogging {
             exceptionFormat = 'full'
         }
+        retry {
+            // Allow for retrying flaky tests.
+            maxRetries = 3
+            maxFailures = 20
```

Comment:
Does this mean total number of tests failed during the build across all modules? Maybe it might be a good idea to comment that.